src/components/WalletConnect.vue

```diff
@@ -106,12 +106,19 @@ import {
 } from "@algo-builder/web";
 import WalletStore from "../store/WalletStore";
 import { searchApplicationAndAccount } from "@/indexer";
-import { walletMessage } from "../constants/constant";
+import {
+	MAIN_NET_URL,
+	BETA_NET_URL,
+	TEST_NET_URL,
+	walletMessage,
+} from "../constants/constant";
 import DaoID from "@/store/DaoID";
 import {
 	openErrorNotificationWithIcon,
 	WALLET_CONNECTION_ERROR,
 } from "@/constants";
```

Comment:
What is different between `../constants/constant` and `@constants` ?

Reply:
`@constants` tells the folder from which to import i.e it's `index.tx` while `../constants/constant` is the specific address of the file